crawl-ref/source/player-stats.cc

```diff
@@ -690,3 +691,132 @@ bool have_stat_zero()
 
     return false;
 }
+
+/*Gnoll ability to shift three points from one attribute to another,
+ *becoming drained if successful.
```

Comment:
Should have spaces after each `*`, and not have text on the same line as the opening `/*`.

Reply:
Fixed and rebased into the Shift Attributes commit.